common/flatpak-transaction.c

```diff
@@ -1275,9 +1275,8 @@ flatpak_transaction_class_init (FlatpakTransactionClass *klass)
    * ref.
    *
    * If the caller wants to install the rebased ref, they should call
-   * flatpak_transaction_add_uninstall() on @ref,
-   * flatpak_transaction_add_rebase() on @rebased_to_ref, and return %TRUE.
-   * Otherwise %FALSE may be returned.
+   * flatpak_transaction_add_rebase_and_uninstall() on @ref and @rebased_to_ref,
```

Comment:
```suggestion
   * flatpak_transaction_add_rebase_and_uninstall() on @rebased_to_ref and @ref,
```

if you take my other suggestion